internal/pkg/cli/init_test.go

```diff
@@ -168,6 +168,10 @@ func TestInitOpts_Run(t *testing.T) {
 						Value: manifestinfo.WorkerServiceType,
 						Hint:  "Events to SQS to ECS on Fargate",
 					},
+					{
+						Value: manifestinfo.StaticSiteType,
+						Hint: "Internet to CDN to S3 bucket",
```

Comment:
I might have missed something but I just want to make sure: is there always always a CDN in front?

Reply:
For now yes 🥺